source/develop/troubleshooting-nfs-storage-issues.md

```diff
@@ -106,6 +107,21 @@ A new **nfs-check** script is now available to test whether an NFS export is rea
 
      #>/etc/init.d/nfs-kernel-server restart
 ```
+#### Ubuntu 22.04 (Jammy)
+In addition to what is above you will need to edit the file /etc/nfs.conf and comment out manage-gids=y
+```console
+      [mountd]
+      # manage-gids=y
+ ```
+ Edit /etc/default/nfs-kernel-server and remove --manage-gids from RPCMOUNTDOPTS="--manage-gids"
```

Comment:
same here with "\`/etc/default/nfs-kernel-server\`" and "\`--manage-gids from RPCMOUNTDOPTS="--manage-gids"\`"